sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Spec/CloudTask.json

```diff
@@ -336,6 +336,17 @@
         "UnboundAccess": "read,write"
       },
       "Value": null
+    },
+    {
+      "Key": {
+        "Type": "int?",
+        "Name": "RequiredSlots",
+        "SummaryComment": "The number of scheduling slots that the Task required to run.",
+        "RemarksComment": "The default is 1. A Task can only be scheduled to run on a compute node if the node has enough free scheduling slots available. For multi-instance Tasks, this must be 1.",
```

Comment:
What if it's not 1? Does it default to 1? Will it fail?

Reply:
The task add will fail once it hits the FE